index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>My Portfolio</title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  </head>
+
+  <body>
+    <header class="header">
+        <a href="#" class="logo">My Logo</a>
+        <nav id="nav-hamburger">
+            <label class="ham-icon" for="hamburger">&#9776;</label>            
+        </nav>
+
+        <nav class="nav">
+        <ul class="menu">            
+            <li><a href="#" class="nav-item">Portfolio</a></li>
+            <li><a href="#" class="nav-item">About</a></li>
+            <li><a href="#" class="nav-item">Contact</a></li>            
+        </ul>
+        </nav>
+    </header>
+    <section id="main">
+        <h2 class="main-heading">I'm Misal. Glad to see you!</h2>
+        <p class="main-paragraph">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+    </section>
+
```

Comment:
- [ ] Our design should be identical to the figma design as per requirements. so kindly consider adding `Let's connect` and then social icons here. social icons should be taken as `a` tag in `li` in `ul`.
your design
![re](https://user-images.githubusercontent.com/96683843/226989298-f2eceba0-397d-4c0c-a374-3c3228580563.jpg)

figma design
![qw](https://user-images.githubusercontent.com/96683843/226989332-4df747e6-f1b0-4aae-bbdc-fd4d98f16a19.jpg)